react/classes/react-dynamic-data-workshop/src/components/UserHeader.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import { getUserData as UserData } from '../utils/getUserData.js';
+import { token } from '../../token';
 
 export default class UserHeader extends Component {
   constructor(props) {
```

Comment:
you do have a lonely constructor out here :smirk:

Reply:
thank you <3 
no lonely constructors allowed here 
I'll make sure to give him what he deserve